### ISSUE TITLE
Broken FN token

### ISSUE THREAD

**Issue by kukulich** [squizlabs/PHP_CodeSniffer#2940]
```php
	public function anything($list)
	{
		return count(array_filter($list->call('something'), fn (object $object): bool => $object->getAnything() === $this->anything)) === 1;
	}
```

`)` in `$object->getAnything()` is reported as `scope_closer` but it's wrong.

**Comment by rodrigoprimo** [comment]
The arrow function `scope_closer` tokenization bug reported here appears to be fixed. On version 3.5.5, as mentioned in the issue description, the `scope_closer` for the `T_FN` token in the code sample incorrectly points to the `)` in `$object->getAnything()`.

I just tested and confirmed that on the latest PHPCS version, it correctly points to the last token of the arrow function expression (`anything`).

I believe this issue can be closed.
